pkg/controller/grouppermission/grouppermission_controller.go

```diff
@@ -203,21 +203,22 @@ func populateCrClusterRoleNames(groupPermission *managedv1alpha1.GroupPermission
 }
 
 // populateClusterRoleBindingNames to see if ClusterRoleBinding exists in k8s ClusterRoleBindlingList
-// returns bool
-func populateClusterRoleBindingNames(clusterRoleBindingName []string, clusterRoleBindingList *v1.ClusterRoleBindingList) []string {
+// returns a slice of clusterRoleBindingNames that exists in CR but not in clusterRoleBindingList
+func populateClusterRoleBindingNames(clusterRoleBindingNames []string, clusterRoleBindingList *v1.ClusterRoleBindingList) []string {
 	var crClusterRoleBindingList []string
+	var found bool
```

Comment:
```suggestion
	found := false
```

It may be safer to initialize the value?

Reply:
@jharrington22 default for bool is false I believe.